src/test/java/com/jfrog/ide/common/yarn/YarnDriverTest.java

```diff
@@ -0,0 +1,91 @@
+package com.jfrog.ide.common.yarn;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.io.FileUtils;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.testng.Assert.*;
+
+/**
+ * Test the 'why' functionality in YarnDriver.
+ */
+public class YarnDriverTest {
+    private static final Path YARN_ROOT = Paths.get(".").toAbsolutePath().normalize().resolve(Paths.get("src", "test", "resources", "yarn"));
+
+    enum Project {
```

Comment:
Help to resolve the "Class 'Project' is exposed outside its defined visibility scope" warning
```suggestion
    public enum Project {
```

Same comment for the Project in YarnTreeBuilderTest